src/ast/codegen_llvm.cpp

```diff
@@ -1499,24 +1516,46 @@ AllocaInst *CodegenLLVM::getMapKey(Map &map)
 {
   AllocaInst *key;
   if (map.vargs) {
-    size_t size = 0;
-    for (Expression *expr : *map.vargs)
+    if (map.vargs->size() == 1)
```

Comment:
I think this could do with a small comment. The goal of this is to avoid a useless copy when we only have one key for a map

Reply:
Added these comments (sorry about the squashed diff)

https://github.com/iovisor/bpftrace/blob/b936d524b5c1d33efe555ec69b5d7b92eccc8c16/src/ast/codegen_llvm.cpp#L1544
https://github.com/iovisor/bpftrace/blob/b936d524b5c1d33efe555ec69b5d7b92eccc8c16/src/ast/codegen_llvm.cpp#L1552
https://github.com/iovisor/bpftrace/blob/b936d524b5c1d33efe555ec69b5d7b92eccc8c16/src/ast/codegen_llvm.cpp#L1565
https://github.com/iovisor/bpftrace/blob/b936d524b5c1d33efe555ec69b5d7b92eccc8c16/src/ast/codegen_llvm.cpp#L1574
https://github.com/iovisor/bpftrace/blob/b936d524b5c1d33efe555ec69b5d7b92eccc8c16/src/ast/codegen_llvm.cpp#L1600